### ISSUE TITLE
[DataCap Application] USC Shoah Foundation

### ISSUE THREAD

**Issue by galen-mcandrew** [filecoin-project/filecoin-plus-large-datasets#53]
# Large Dataset Notary Application

To apply for a DataCap allocation for your dataset, please fill out the following information.

## Core Information
- Organization Name: USC Shoah Foundation
- Website / Social Media: http://sfi.usc.edu/ & https://www.youtube.com/user/USCShoahFoundation
- Total amount of DataCap being requested (between 500 TiB and 5 PiB): 5 PiB
- Weekly allocation of DataCap requested (usually between 1-100TiB): 100TiB
- On-chain address for first allocation:  f17g7h52bsi53rb263xwne573dusskit4mieqkgry

_Please respond to the questions below in pargraph form, replacing the text saying "Please answer here". Include as much detail as you can in your answer!_

## Project details

Share a brief history of your project and organization.
```
USC Shoah Foundation – The Institute for Visual History and Education develops empathy, understanding and respect through testimony, using its Visual History Archive of more than 55,000 video testimonies, award-winning IWitness education program, and the Center for Advanced Genocide Research. USC Shoah Foundation's interactive programming, research and materials are accessed in museums and universities, cited by government leaders and NGOs, and taught in classrooms around the world. Now in its third decade, USC Shoah Foundation reaches millions of people on six continents from its home at the Dornsife College of Letters, Arts and Sciences at the University of Southern California.
```

What is the primary source of funding for this project?
```
Filecoin Foundation for the Decentralized Web (FFDW)
```

What other projects/ecosystem stakeholders is this project associated with?
```
Starling Labs
```

## Use-case details

Describe the data being stored onto Filecoin
```
Digital Library of Survivor Testimonies - compilation of audiovisual content from holocaust and genocide survivors. Majority of the data is lossless copies of collected data, but some of the dataset is lower quality replicas of the content.
```

Where was the data in this dataset sourced from?
```
Audiovisual content was recorded by volunteers and is stored on tape drives at the University of Southern California. It consists of live interviews with survivors of holocaust and genocide. Maintaining the integrity of the original content is extremely important, and USC constantly runs fixity checks on the content. 
```

Can you share a sample of what is in the dataset? A link to a file, an image, a table, etc., are good examples of this. 
```
A set of testimonies is available on YouTube and viewable by anyone: https://www.youtube.com/playlist?list=PLWIFgIFN2QqiDdkA-MXpsvZOSvTYkEGsL. 
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
A lot of the data is already publicly available for view via the Visual History Archive (https://vhaonline.usc.edu/). Some of the data has been requested to be private for a period of time based on requests from the interviewees of the content. 
```

What is the expected retrieval frequency for this data?
```
This is primarily for long-term archiving purposes only. Viewing / retrieving of this content should primarily be happening through the Visual History Archive (https://vhaonline.usc.edu/).
```

For how long do you plan to keep this dataset stored on Filecoin? Will this be a permanent archival or a one-time storage deal?
```
Yes, permanent archival. 
```

## DataCap allocation plan

In which geographies do you plan on making storage deals?
```
We will be prioritizing making deals globally in any geography where our content can be legally stored by storage providers. 
```

What is your expected data onboarding rate? How many deals can you make in a day, in a week? How much DataCap do you plan on using per day, per week?
```
The current plan is to use offline data transfer mechanisms that will enable 100s of terabytes of content to be stored on a weekly basis. We hope to have access to at least 100TiB of DataCap per week.
```

How will you be distributing your data to miners? Is there an offline data transfer process?
```
Yes, there is going to be an offline data transfer process either through hosting files online where storage providers can download them or (where logistically feasible) through the distribution of content on physical drives. 
```

How do you plan on choosing the miners with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
We would like to work with several reputable large-scale storage provider operations to ensure geo-distribution and reliability of storage.  
```

How will you be distributing data and DataCap across miners storing data?
```
This project aims to onboard 4+PiB of original data, for which we’d like to store multiple replicas (2-5) with separate storage providers for each replica. Deals will be structured to be as close to sector size as possible for a storage provider. 
```

**Comment by galen-mcandrew** [comment]
@starling-admin Here is the new large dataset application issue, per the new LDN process.

**Comment by galen-mcandrew** [comment]
## Multisig Notary requested
#### Total DataCap requested 
> 5PiB
#### Expected weekly DataCap usage rate 
> 100TiB

**Comment by starling-admin** [comment]
@dkkapur + the Datacap notaries, 

We'd like to move our next tranche of our approved data cap allocation to this wallet: f17g7h52bsi53rb263xwne573dusskit4mieqkgry

We are actively sealing deals and ready to scale.

Thanks!

**Comment by galen-mcandrew** [comment]
According to glif, address f3w5fx6wta4ewl2iyf7xcogmzffz2fmrngpzdpduj3xmk3dwjxc6dyq36gdf3rflkkrblh5nci5xymc5hal3qq has 14 TiB remaining. With the initial allocation of 50TiB, that means the next allocation should kick off at ~12 TiB. 

Checking lotus, seeing 2 pending transactions with only 1 approval, so I want to check in with the team before I make any changes to the client address.

@fabriziogianni7 @ialberquilla

**Comment by galen-mcandrew** [comment]
Additionally flagging for notaries:
@starling-admin according to a quick audit (https://filplus.d.interplanetary.one/clients/f0700600/breakdown), while you have worked with 20 storage providers, an extreme majority of your deal-making has been with a single storage provider (**84%**). 

Per your application and the large dataset process, can you provide some more details about your deal distribution plan?

**Comment by starling-admin** [comment]
Hi Galen,

We are still testing the deal automation with Bitbot so we are
progressively decentralizing. The plan so far is as follows:

Pilot 81 TiBs sealed with 1 provider (as a test)
Pilot 81 TiBs sealed with 4 providers
Pilot 81 TiBs sealed with 10 providers

With these pilots finished we are then going to work on a plan for 4 PiBs.
The plan there is to distribute to an initial set of 10 providers and then
scale to as many as possible.
The fulcrum will be what type of financing SPs will need to take on to meet
their pledge. We are working with PL on that mechanism.

Happy to answer any questions you have.

Best,
Jonathan

**Comment by galen-mcandrew** [comment]
Edited parent comment and the request comment with new client address, app front-end appears to be displaying correctly.
![Screen Shot 2021-12-17 at 9 57 23 AM](https://user-images.githubusercontent.com/85587348/146587803-b1ea7171-615b-49b3-b80e-e7896d20fa6e.png)

**Comment by MegTei** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceclyqpuoqqnnp6bzyanmlee53jppwti5cvggs556im25qqtb5mrj4 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 100TiB
#### Signer Address
> f1ystxl2ootvpirpa7ebgwl7vlhwkbx2r4zjxwe5i
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceclyqpuoqqnnp6bzyanmlee53jppwti5cvggs556im25qqtb5mrj4

**Comment by Ray-PiKNiK** [comment]
Hi @MegTei,

It seems the network still is not aware of `f17g7h52bsi53rb263xwne573dusskit4mieqkgry` being a verified client. It is unable to send verified deals and https://verify.glif.io/ reflects this state as well.

Any ideas what could be the issue?

**Comment by dkkapur** [comment]
@Ray-PiKNiK - we need one more notary to sign before this gets allocated. 2 signatures from participating notaries are required for each allocation to go through.

**Comment by Ray-PiKNiK** [comment]
Dear Datacap Notaries,

We are getting ready to ramp up deal distribution and would like to start the data cap tranche allocation process now in order to reduce the turnaround time needed moving forward.

Thanks!

**Comment by starling-admin** [comment]
Dear Datacap Notaries,

We are all excited to start scaling our implementation with the USC Shoah Foundation dataset. This will involve 2 key changes: 

1. We are working now to spread the data to 4 SP (to meet the Datacap base decentralization requirement) and identified a total of 10 SPs beyond PikNik that we want to store data with. We are in the process of sending test data sets to each and are very motivated to get the data distributed as far and wide as possible.

2. To help us keep pace, I am deputizing @Ray-PiKNiK to handle our interactions with you going forward. This includes troubleshooting these DataCap issues and coming to you with requests to manually release FIL+ allocations

Thanks again for understanding!

**Comment by MegTei** [comment]
HI @Ray-PiKNiK did you need anything more from me at this stage?

**Comment by Ray-PiKNiK** [comment]
Hi @MegTei, we plan to start distributing the deals next week. The aggregate sealing rate of the storage providers will be at least 25 TiBs/day. With the remaining data cap available (50T) on our wallet, we would run out in 2 days.

While the allocation bot will get trigger at <25% available, we would like to preemptively request the next allocation of data cap now in preparation for next week if possible.

Thank you!

**Comment by flyworker** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedafmdb6knk72czet55h7d6uwq3c2k6cda55wkppsp4wsrrkpmhyo 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 200.00TiB
#### Signer Address
> f1hlubjsdkv4wmsdadihloxgwrz3j3ernf6i3cbpy
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedafmdb6knk72czet55h7d6uwq3c2k6cda55wkppsp4wsrrkpmhyo

**Comment by MegTei** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacednk4zf5l4gu4gwz3q76vsr5wyefusrbxhypwtu6q7uaqgbcbzacs 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 200.00TiB
#### Signer Address
> f1ystxl2ootvpirpa7ebgwl7vlhwkbx2r4zjxwe5i
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacednk4zf5l4gu4gwz3q76vsr5wyefusrbxhypwtu6q7uaqgbcbzacs

**Comment by cryptowhizzard** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceb5kexkgrjffzo4h4nrccv5ziqhafajikg5rxiul6eygwatv2dny4 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 400.00TiB
#### Signer Address
> f1krmypm4uoxxf3g7okrwtrahlmpcph3y7rbqqgfa
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceb5kexkgrjffzo4h4nrccv5ziqhafajikg5rxiul6eygwatv2dny4

**Comment by dannyob** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedkbtept24dczb7p2jpcsrwbvpdwvb3fy5vd5aa6dvpjl45arrsoe 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 400.00TiB
#### Signer Address
> f1k6wwevxvp466ybil7y2scqlhtnrz5atjkkyvm4a
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedkbtept24dczb7p2jpcsrwbvpdwvb3fy5vd5aa6dvpjl45arrsoe

**Comment by Yvette516** [comment]
> @dkkapur + the Datacap notaries,
> 
> We'd like to move our next tranche of our approved data cap allocation to this wallet: f17g7h52bsi53rb263xwne573dusskit4mieqkgry
> 
> We are actively sealing deals and ready to scale.
> 
> Thanks!

Could you provide your SP node and rootcid ? We need to retrieve your public dataset.

**Comment by jamerduhgamer** [comment]
Hi @Yvette516, unfortunately the majority of dataset will be unable to retrieved as mentioned in datacap request that "some of the data has been requested to be private for a period of time based on requests from the interviewees of the content." The data will be made public after enough time has passed depending on USC SFI's decision.

You can see some samples of the files here - https://vhaonline.usc.edu/
Some of the testimonies here - https://www.youtube.com/playlist?list=PLWIFgIFN2QqiDdkA-MXpsvZOSvTYkEGsL.

**Comment by Yvette516** [comment]
@jamerduhgamer @dkkapur Before you said that the data you provided could be publicly accessible, but now it can't. You violated the rules of the whole community.

**Comment by stuberman** [comment]
> Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
> 
> ```
> A lot of the data is already publicly available for view via the Visual History Archive (https://vhaonline.usc.edu/). Some of the data has been requested to be private for a period of time based on requests from the interviewees of the content. 
> ```

I do not see where it was claimed that the data set would be public, here is the actual question and response:

`Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).`

> A lot of the data is already publicly available for view via the Visual History Archive (https://vhaonline.usc.edu/). Some of the data has been requested to be private for a period of time based on requests from the interviewees of the content. 

Having worked directly with the Shoah Foundation and this project for the last year, it was clearly articulated that this data set is not public and needs to be kept private to protect the privacy and lives of those who survived the genocide attempt and recorded their testimonies for posterity.

**Comment by cryptowhizzard** [comment]
I think there is a miscommunication going on @Yvette516 

PiKNiK send this data to different SP’s across the globe and you can retrieve that data. What is being referred to is the dataset itself. A small part of it is not send to SP’s for public use to store and have retrievable. 

From the deals we (Dcent) received from PikNik you are welcome to retrieve them!

**Comment by MegTei** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceajdkmdf3vhw2t474mqs535rmemcammen7pumh7rxp47csi7bh3uy 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1ystxl2ootvpirpa7ebgwl7vlhwkbx2r4zjxwe5i
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceajdkmdf3vhw2t474mqs535rmemcammen7pumh7rxp47csi7bh3uy

**Comment by Yvette516** [comment]
As @Sunnyiscoming pointed out in https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/420#issuecomment-1173411027, a single SP was allocating with an extremely high DataCap which is not a good way to backup data and violating Filecoin rules. While @Ray-PikNik did make an explanation but it was a rogue one. Yeah "in the long term" each SP will have an even share, but 16 days since the explanation, nothing has changed, f01833311 still holds over 62% of total DataCap. So if "long term" means a decade or even a century, that absolutely ridiculous. Community rules for everyone will make no sense. 
Besides, I requested to have rootCIDs but was refused since "some of the data has been requested to be private for a period of time based on requests from the interviewees of the content.". Again they need "a period of time", a rogue content that make no sense at all. We need a roadmap, a precise one. Leaving this behind, "some of the data is private", so we do have some rest of the data that are publicly retrievable aren't they? Why can't you provide rootCIDs for supervision? 
To sum up,  I propose to re-evaluate this application and concerning issue #420 until a detailed, precise and reasonable explanation is given.

**Comment by kernelogic** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebhkixh3tp4zqmhwdx5hog2helm3fmekutwo2u22jen6uh5jkfdhe 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebhkixh3tp4zqmhwdx5hog2helm3fmekutwo2u22jen6uh5jkfdhe

**Comment by dannyob** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacea3kfmzkfvcwhhm7s2ljtwyv7jcadsjbtd5nrtr3hgwzlpu2wemgo 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1k6wwevxvp466ybil7y2scqlhtnrz5atjkkyvm4a
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacea3kfmzkfvcwhhm7s2ljtwyv7jcadsjbtd5nrtr3hgwzlpu2wemgo

**Comment by Yvette516** [comment]
> As @Sunnyiscoming pointed out in [#420 (comment)](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/420#issuecomment-1173411027), a single SP was allocating with an extremely high DataCap which is not a good way to backup data and violating Filecoin rules. While @Ray-PiKNiK did make an explanation but it was a rogue one. Yeah "in the long term" each SP will have an even share, but 16 days since the explanation, nothing has changed, f01833311 still holds over 62% of total DataCap. So if "long term" means a decade or even a century, that absolutely ridiculous. Community rules for everyone will make no sense. Besides, I requested to have rootCIDs but was refused since "some of the data has been requested to be private for a period of time based on requests from the interviewees of the content.". Again they need "a period of time", a rogue content that make no sense at all. We need a roadmap, a precise one. Leaving this behind, "some of the data is private", so we do have some rest of the data that are publicly retrievable aren't they? Why can't you provide rootCIDs for supervision? To sum up, I propose to re-evaluate this application and concerning issue #420 until a detailed, precise and reasonable explanation is given.

@starling-admin just want remind you it's been over a month now and I'm still waiting for your answers. 
![image](https://user-images.githubusercontent.com/106008819/187659641-6646843d-d1c7-498e-a8f2-e7c7593c2a53.png)

f01833311 still holds 46% of the total allocation. This is extremely high beyond what we generally consider to be a common range. With such a high percentage (which was even higher before), I don't understand why you didn't disclose more detail about this SP. This's not how transparency should be. Also @galen-mcandrew the latest two rounds of multisig were done by @kernelogic. This is a clear violation of community rules.

**Comment by jamerduhgamer** [comment]
Hello @Yvette516, we have stopped sending deals to f01833311 since they are slightly higher than the 30% requirement. You can check on Filfox here - https://filfox.info/en/address/f01833311 and this SP has not grown in power since July 31st, 7/31.

It seems like a bug that Kernelogic was able to approve two rounds of multisig in a row. We should implement a process that does not allow this to happen.

**Comment by kernelogic** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebnblgwsm2vov6dadqpz3t6d2rjxkx4rii4op4zwxgmo3x36m2tms 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### Id
> a8b70877-afe7-49e6-87c0-17ff7244b5a5
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebnblgwsm2vov6dadqpz3t6d2rjxkx4rii4op4zwxgmo3x36m2tms

**Comment by dannyob** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecm6skzg52gmaffahaercg2emcbcvkjgvgcweminphls3xrh3sqjs 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1k6wwevxvp466ybil7y2scqlhtnrz5atjkkyvm4a
#### Id
> a8b70877-afe7-49e6-87c0-17ff7244b5a5
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecm6skzg52gmaffahaercg2emcbcvkjgvgcweminphls3xrh3sqjs

**Comment by filplus-checker** [comment]
## DataCap and CID Checker Report[^1]
 - Organization: `USC Shoah Foundation`
 - Client: `f17g7h52bsi53rb263xwne573dusskit4mieqkgry`
### Storage Provider Distribution
The below table shows the distribution of storage providers that have stored data for this client.

If this is the first time a provider takes verified deal, it will be marked as `new`.

For most of the datacap application, below restrictions should apply.
 - Storage provider should not exceed 25% of total datacap.
 - Storage provider should not be storing duplicate data for more than 20%.
 - Storage provider should have published its public IP address.
 - All storage providers should be located in different regions.

⚠️ [f01833311](https://filfox.info/en/address/f01833311) has sealed 25.73% of total datacap.

| Provider                                                    |                         Location | Total Deals Sealed | Percentage | Unique Data | Duplicate Deals |
| :---------------------------------------------------------- | -------------------------------: | -----------------: | ---------: | ----------: | --------------: |
| [f01833311](https://filfox.info/en/address/f01833311)`new`  |            Las Vegas, Nevada, US |         622.34 TiB |     25.73% |  599.06 TiB |           3.74% |
| [f01833348](https://filfox.info/en/address/f01833348)`new`  |      Sydney, New South Wales, AU |         406.16 TiB |     16.79% |  399.31 TiB |           1.69% |
| [f01840436](https://filfox.info/en/address/f01840436)`new`  |            Lincoln, Nebraska, US |         385.44 TiB |     15.93% |  377.10 TiB |           2.16% |
| [f01872281](https://filfox.info/en/address/f01872281)`new`  |            Las Vegas, Nevada, US |         286.88 TiB |     11.86% |  271.28 TiB |           5.44% |
| [f01880776](https://filfox.info/en/address/f01880776)`new`  |            Las Vegas, Nevada, US |         232.31 TiB |      9.60% |  229.03 TiB |           1.41% |
| [f01858235](https://filfox.info/en/address/f01858235)`new`  |             Denver, Colorado, US |         199.34 TiB |      8.24% |  198.59 TiB |           0.38% |
| [f01886706](https://filfox.info/en/address/f01886706)`new`  |            Las Vegas, Nevada, US |          96.75 TiB |      4.00% |   95.91 TiB |           0.87% |
| [f01832393](https://filfox.info/en/address/f01832393)`new`  |          Seattle, Washington, US |          69.91 TiB |      2.89% |   69.88 TiB |           0.04% |
| [f01864397](https://filfox.info/en/address/f01864397)`new`  |           Zaventem, Flanders, BE |          52.63 TiB |      2.18% |   52.28 TiB |           0.65% |
| [f01206283](https://filfox.info/en/address/f01206283)`new`  |      Sydney, New South Wales, AU |          26.53 TiB |      1.10% |   26.38 TiB |           0.59% |
| [f01652333](https://filfox.info/en/address/f01652333)       |        Sunnyvale, California, US |          17.53 TiB |      0.72% |   17.53 TiB |           0.00% |
| [f01558688](https://filfox.info/en/address/f01558688)       |             Toronto, Ontario, CA |          16.20 TiB |      0.67% |   16.20 TiB |           0.00% |
| [f01207045](https://filfox.info/en/address/f01207045)       | Heerhugowaard, North Holland, NL |           2.50 TiB |      0.10% |    2.50 TiB |           0.00% |
| [f01201327](https://filfox.info/en/address/f01201327)       | Heerhugowaard, North Holland, NL |           1.63 TiB |      0.07% |    1.63 TiB |           0.00% |
| [f01208862](https://filfox.info/en/address/f01208862)       | Heerhugowaard, North Holland, NL |         928.00 GiB |      0.04% |  928.00 GiB |           0.00% |
| [f01199430](https://filfox.info/en/address/f01199430)       | Heerhugowaard, North Holland, NL |         800.00 GiB |      0.03% |  800.00 GiB |           0.00% |
| [f01199442](https://filfox.info/en/address/f01199442)       | Heerhugowaard, North Holland, NL |         640.00 GiB |      0.03% |  640.00 GiB |           0.00% |
| [f01969933](https://filfox.info/en/address/f01969933)`new`  |      Sydney, New South Wales, AU |         160.00 GiB |      0.01% |  160.00 GiB |           0.00% |
| [f01776738](https://filfox.info/en/address/f01776738)       |             Montréal, Quebec, CA |          65.00 GiB |      0.00% |   65.00 GiB |           0.00% |
| [f01746964](https://filfox.info/en/address/f01746964)`new`  |      Sydney, New South Wales, AU |          65.00 GiB |      0.00% |   65.00 GiB |           0.00% |
| [f01775922](https://filfox.info/en/address/f01775922)       |            Ashburn, Virginia, US |          65.00 GiB |      0.00% |   65.00 GiB |           0.00% |
| [f019104](https://filfox.info/en/address/f019104)           |             Montréal, Quebec, CA |          64.00 GiB |      0.00% |   64.00 GiB |           0.00% |
| [f010446](https://filfox.info/en/address/f010446)           |           Zaventem, Flanders, BE |          64.00 GiB |      0.00% |   64.00 GiB |           0.00% |

![Provider Distribution](https://raw.githubusercontent.com/data-preservation-programs/filplus-checker-assets/main/filecoin-project/filecoin-plus-large-datasets/issues/53/1671091876875.png)
### Deal Data Replication
The below table shows how each many unique data are replicated across storage providers.
- No more than 25% of unique data are stored with less than 4 providers.

⚠️ 99.97% of deals are for data replicated across less than 4 storage providers.

| Unique Data Size | Total Deals Made | Number of Providers | Deal Percentage |
| ---------------: | ---------------: | ------------------: | --------------: |
|       807.74 TiB |       826.30 TiB |                   1 |          34.16% |
|       775.50 TiB |         1.55 PiB |                   2 |          65.81% |
|         1.00 GiB |         3.00 GiB |                   3 |           0.00% |
|        32.00 GiB |       416.00 GiB |                  11 |           0.02% |
|        32.00 GiB |       416.00 GiB |                  12 |           0.02% |

![Replication Distribution](https://raw.githubusercontent.com/data-preservation-programs/filplus-checker-assets/main/filecoin-project/filecoin-plus-large-datasets/issues/53/1671091877553.png)
### Deal Data Shared with other Clients
The below table shows how many unique data are shared with other clients.
Usually different applications owns different data and should not resolve to the same CID.

⚠️ CID sharing has been observed.

| Other Client                                                                                                                                                                                                              | Application                                                                                                                           | Total Deals Affected | Unique CIDs | Verifier |
| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :------------------------------------------------------------------------------------------------------------------------------------ | -------------------: | ----------: | -------: |
| [f3w5fx6wta4ewl2iyf7xcogmzffz2fmrngpzdpduj<br/>3xmk3dwjxc6dyq36gdf3rflkkrblh5nci5xymc5ha<br/>l3qq](https://filfox.info/en/address/f3w5fx6wta4ewl2iyf7xcogmzffz2fmrngpzdpduj3xmk3dwjxc6dyq36gdf3rflkkrblh5nci5xymc5hal3qq) | [USC Shoah Foundation\[DEPRECATED\] USC Shoah Foundation](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/27) |            41.98 TiB |       1,282 |   LDN 27 |
| [f144zep4gitj73rrujd3jw6iprljicx6vl4wbeavi](https://filfox.info/en/address/f144zep4gitj73rrujd3jw6iprljicx6vl4wbeavi)                                                                                                     | [Textile](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/61)                                                 |             3.16 TiB |          36 | LDN # 61 |

[^1]: To manually trigger this report, add a comment with text `checker:manualTrigger`

**Comment by Sunnyiscoming** [comment]
Hi, please explain the abnormal information.

**Comment by kernelogic** [comment]
@Sunnyiscoming are you referring to the CID Sharing or something else?

**Comment by jamerduhgamer** [comment]
Hi @Sunnyiscoming, `f3w5fx6wta4ewl2iyf7xcogmzffz2fmrngpzdpduj3xmk3dwjxc6dyq36gdf3rflkkrblh5nci5xymc5hal3qq` was also PiKNiK and was a proof of concept for the dataset.

The 3.16 TiBs from Textile was when we tested sending out the data via Bidbot a long time ago.

**Comment by cryptowhizzard** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebiratoulbpgqeqrpsd75s2fwed7k6qivcvyq7eylqp52jrxxn7po 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1krmypm4uoxxf3g7okrwtrahlmpcph3y7rbqqgfa
#### Id
> 32d5de4c-fdcf-4cea-b2c3-72e3f7cac4c4
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebiratoulbpgqeqrpsd75s2fwed7k6qivcvyq7eylqp52jrxxn7po

**Comment by jamerduhgamer** [comment]
@simonkim0515, @dkkapur, is the above message an issue?

**Comment by flyworker** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedp3oncietykyegldaguooamtcnxxefqvwfukxhbfko63f7t44lvy 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1hlubjsdkv4wmsdadihloxgwrz3j3ernf6i3cbpy
#### Id
> 32d5de4c-fdcf-4cea-b2c3-72e3f7cac4c4
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedp3oncietykyegldaguooamtcnxxefqvwfukxhbfko63f7t44lvy

**Comment by xinaxu** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacea5f2v3xtlbrvu5zapqnzzq7sshbmguelu5imkvbr2j6hthov7qnc 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1k3ysofkrrmqcot6fkx4wnezpczlltpirmrpsgui
#### Id
> 
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacea5f2v3xtlbrvu5zapqnzzq7sshbmguelu5imkvbr2j6hthov7qnc

**Comment by kernelogic** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecwqx75of5p5w3r7gk7o5xfbptug65psqga6keqwsbkvciscytl2u 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### Id
> 
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecwqx75of5p5w3r7gk7o5xfbptug65psqga6keqwsbkvciscytl2u

**Comment by cryptowhizzard** [comment]
checker:manualTrigger

**Comment by cryptowhizzard** [comment]
PiKNiK, in collaboration with USC Shoah, kindly requested our attention to review and sign the LDN. Their data exhibits exceptional value and holds significance for the ecosystem. To ensure wider acceptance of the CID report, it is recommended to distribute it to one additional global service provider.

**Comment by cryptowhizzard** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedwvpuzrvf4eunpl47cwccck3hd5ovouxxylxjgwtv4o45nwejyg4 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 400.00TiB
#### Signer Address
> f1krmypm4uoxxf3g7okrwtrahlmpcph3y7rbqqgfa
#### Id
> 6fad9f7c-2f64-43fb-826c-c8515a728dac
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedwvpuzrvf4eunpl47cwccck3hd5ovouxxylxjgwtv4o45nwejyg4

**Comment by xinaxu** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecaamvtmuetlnrxxond3mmdhjvn4ivi4rbcjd42wprkq6pt6cc5xc 
#### Address 
> f17g7h52bsi53rb263xwne573dusskit4mieqkgry
#### Datacap Allocated
> 400.00TiB
#### Signer Address
> f1k3ysofkrrmqcot6fkx4wnezpczlltpirmrpsgui
#### Id
> 6fad9f7c-2f64-43fb-826c-c8515a728dac
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecaamvtmuetlnrxxond3mmdhjvn4ivi4rbcjd42wprkq6pt6cc5xc
